res/text/meson.build

```diff
@@ -1,739 +1,749 @@
+    'pl_msg_0608.gmm',
+    'pl_msg_0609.gmm',
+    'pl_msg_0610.gmm',
+    'pl_msg_0611.gmm',
+    'pl_msg_0612.gmm',
+    'pl_msg_0613.gmm',
+    'pl_msg_0614.gmm',
+    'pl_msg_0615.gmm',
+    'pl_msg_0616.gmm',
+    'pl_msg_0617.gmm',
+    'pl_msg_0618.gmm',
+    'pl_msg_0619.gmm',
+    'pl_msg_0620.gmm',
+    'pl_msg_0621.gmm',
+    'pl_msg_0622.gmm',
+    'pl_msg_0623.gmm',
+    'pl_msg_0624.gmm',
+    'pl_msg_0625.gmm',
+    'pl_msg_0626.gmm',
+    'pl_msg_0627.gmm',
+    'pl_msg_0628.gmm',
+    'pl_msg_0629.gmm',
+    'pl_msg_0630.gmm',
+    'pl_msg_0631.gmm',
+    'pl_msg_0632.gmm',
+    'pl_msg_0633.gmm',
+    'pl_msg_0634.gmm',
+    'pl_msg_0635.gmm',
+    'pl_msg_0636.gmm',
+    'pl_msg_0637.gmm',
+    'pl_msg_0638.gmm',
+    'pl_msg_0639.gmm',
+    'pl_msg_0640.gmm',
+    'pl_msg_0641.gmm',
+    'pl_msg_0642.gmm',
+    'pl_msg_0643.gmm',
+    'pl_msg_0644.gmm',
+    'pl_msg_0645.gmm',
+    'pl_msg_0646.gmm',
+    'pl_msg_0647.gmm',
+    'pl_msg_0648.gmm',
+    'pl_msg_0649.gmm',
+    'pl_msg_0650.gmm',
+    'pl_msg_0651.gmm',
+    'pl_msg_0652.gmm',
+    'pl_msg_0653.gmm',
+    'pl_msg_0654.gmm',
+    'pl_msg_0655.gmm',
+    'pl_msg_0656.gmm',
+    'pl_msg_0657.gmm',
+    'pl_msg_0658.gmm',
+    'pl_msg_0659.gmm',
+    'pl_msg_0660.gmm',
+    'pl_msg_0661.gmm',
+    'pl_msg_0662.gmm',
+    'pl_msg_0663.gmm',
+    'pl_msg_0664.gmm',
+    'pl_msg_0665.gmm',
+    'pl_msg_0666.gmm',
+    'pl_msg_0667.gmm',
+    'pl_msg_0668.gmm',
+    'pl_msg_0669.gmm',
+    'pl_msg_0670.gmm',
+    'pl_msg_0671.gmm',
+    'pl_msg_0672.gmm',
+    'pl_msg_0673.gmm',
+    'pl_msg_0674.gmm',
+    'pl_msg_0675.gmm',
+    'pl_msg_0676.gmm',
+    'pl_msg_0677.gmm',
+    'pl_msg_0678.gmm',
+    'pl_msg_0679.gmm',
+    'pl_msg_0680.gmm',
+    'pl_msg_0681.gmm',
+    'pl_msg_0682.gmm',
+    'pl_msg_0683.gmm',
+    'pl_msg_0684.gmm',
+    'pl_msg_0685.gmm',
+    'pl_msg_0686.gmm',
+    'pl_msg_0687.gmm',
+    'pl_msg_0688.gmm',
+    'pl_msg_0689.gmm',
+    'pl_msg_0690.gmm',
+    'pl_msg_0691.gmm',
+    'pl_msg_0692.gmm',
+    'pl_msg_0693.gmm',
+    'pl_msg_0694.gmm',
+    'pl_msg_0695.gmm',
+    'pl_msg_0696.gmm',
+    'pl_msg_0697.gmm',
+    'pl_msg_0698.gmm',
+    'pl_msg_0699.gmm',
+    'pl_msg_0700.gmm',
+    'pl_msg_0701.gmm',
+    'pl_msg_0702.gmm',
+    'pl_msg_0703.gmm',
+    'pl_msg_0704.gmm',
+    'pl_msg_0705.gmm',
+    'pl_msg_0706.gmm',
+    'pl_msg_0707.gmm',
+    'pl_msg_0708.gmm',
+    'pl_msg_0709.gmm',
+    'pl_msg_0710.gmm',
+    'pl_msg_0711.gmm',
+    'pl_msg_0712.gmm',
+    'pl_msg_0713.gmm',
+    'pl_msg_0714.gmm',
+    'pl_msg_0715.gmm',
+    'pl_msg_0716.gmm',
+    'pl_msg_0717.gmm',
+    'pl_msg_0718.gmm',
+    'pl_msg_0719.gmm',
+    'pl_msg_0720.gmm',
+    'pl_msg_0721.gmm',
+    'pl_msg_0722.gmm',
+    'pl_msg_0723.gmm',
+)
+
+gmm_header_dir = custom_target('gmm_header_dir',
+    output: 'gmm_header_dir',
+    command: [
+        'sh', '-c', '"$@"',
+        '--',
+        'mkdir', '-p', '../include/constants/text'
```

Comment:
mkdir doesn't work without needing a shell?

Reply:
I tried using `run_command`, but that didn't take.